Makefile

```diff
@@ -42,9 +46,11 @@ run-flask: ## Run flask
 run-celery: ## Run celery
 	. environment.sh && celery \
 		-A run_celery.notify_celery worker \
-		--pidfile="/tmp/celery.pid" \
-		--loglevel=INFO \
-		--concurrency=4
+		--uid=$(shell id -u easuser) \
+		--pidfile=/tmp/celery.pid \
+		--loglevel=WARNING \
+		--concurrency=4 \
+		--hostname=0.0.0.0
```

Comment:
I'm sure this will work, but the documentation implies that this param is only hostname not the IP Address to bind to

Reply:
Yeah i think its used for hostname translation but it was the only way we could get celery communicating out of the container.